common/crypto_sort_int32.h

```diff
@@ -0,0 +1,16 @@
+#ifndef CRYPTO_SORT_INT32_H
+#define CRYPTO_SORT_INT32_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
```

Comment:
This should probably be removed, we don't support C++ and I'm not sure how our linters would interact with this header file